manifests/storage_state_crd.yaml

```diff
@@ -10,70 +12,69 @@ spec:
     plural: storagestates
     singular: storagestate
   scope: Cluster
```

Comment:
suggest also explicitly setting `preserveUnknownFields: false` to ensure an existing persisted CRD migrated from v1beta1 turns on pruning